### PR TITLE
extract zipfile inside zipfile in sample data lesson

### PR DESCRIPTION
**Pull request by sumandari**
Please refer to #1328 

## original file

image

image

## Downloaded file
image

### CODE REVIEW

File: django_project/lesson/views/worksheet.py [hunk 1]
```diff
@@ -218,6 +218,56 @@ def render_to_response(self, context, **response_kwargs):
...
+                external_file_zf = zipfile.ZipFile(zip_data_path)
+                for name in external_file_zf.namelist():
+                    if name.endswith('.zip'):
+                        try:
+                            filebytes = BytesIO(external_file_zf.read(name))
+                            subzip = zipfile.ZipFile(filebytes)
+                            for name_subzip in subzip.namelist():
+                                if name_subzip.endswith('/'):
+                                    continue
+                                if name_subzip.startswith('__MACOSX'):
```
Comment (dimasciput) [inline django_project/lesson/views/worksheet.py:243]: @sumandari what about other hidden files? LGTM for now, I think in the future it's better to create a ignore list file.

**Review by dimasciput — APPROVED** [review]
LGTM, please see my comment

File: django_project/lesson/views/worksheet.py (hunk 1 above)
Comment (sumandari) [reply django_project/lesson/views/worksheet.py:243]: thank you @dimasciput 
I will update this after https://github.com/kartoza/prj.app/pull/1314 reviewed/merged. I will use the same ignore list from it.

Comment (timlinux) [reply django_project/lesson/views/worksheet.py:243]: LGTM thanks

Comment (sumandari) [reply django_project/lesson/views/worksheet.py:243]: I will merge this for now, please see #1382 for ignore list file

**Pull request merged**